suite/tests/client-interface/drreg-test.c

```diff
@@ -334,6 +368,32 @@ GLOBAL_LABEL(FUNCNAME:)
         END_FUNC(FUNCNAME)
 #undef FUNCNAME
 
+#define FUNCNAME test_asm_faultC
+        DECLARE_FUNC_SEH(FUNCNAME)
+GLOBAL_LABEL(FUNCNAME:)
+#ifdef X86
+        PUSH_CALLEE_SAVED_REGS()
+        sub      REG_XSP, FRAME_PADDING /* align */
+        END_PROLOG
+
+        jmp      test6
+        /* Test 6: fault aflags restore */
+     test6:
+        mov      TEST_REG_ASM, DRREG_TEST_6_ASM
+        mov      TEST_REG_ASM, DRREG_TEST_6_ASM
+        nop
+        mov      TEST_REG_ASM, DRREG_TEST_7_ASM
+        nop
+        ud2
+
+        jmp      epilog6
+     epilog6:
+        add      REG_XSP, FRAME_PADDING /* make a legal SEH64 epilog */
+        POP_CALLEE_SAVED_REGS()
+        ret
+#endif
+        END_FUNC(FUNCNAME)
+#undef FUNCNAME
 END_FILE
-/* clang-format on */
 #endif
+/* clang-format on */
```

Comment:
See D's comment below

Reply:
A comment in the *source* file.

Also, the on and off no longer match post-preprocessor: we run this file through the preprocessor and the C part now has an on with no preceding off (while the asm part has an off with no on).  We don't run clang-format at that level so it doesn't really matter but it feels wrong: seems nicer to also move the off before the ifdef.

Reply:
Done